Library/Homebrew/cmd/update.sh

```diff
@@ -378,19 +381,34 @@ EOS
   for DIR in "$HOMEBREW_REPOSITORY" "$HOMEBREW_LIBRARY"/Taps/*/*
   do
     [[ -d "$DIR/.git" ]] || continue
+    cd "$DIR" || continue
+
+    TAP_VAR="$(repo_var "$DIR")"
+    declare PREFETCH_REVISION"$TAP_VAR"="$(read_current_revision)"
```

Comment:
Yeah, I'm looking at that commit right now. Still surprising to me that it didn't catch it without that commit. My apologies for the :+1: as well :(